app/1.0.x/upgrading.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Upgrading guide
```

Comment:
The other page calls it "Upgrade guide"; this one says "Upgrading guide". I think "Upgrade guide" reads better.